SUI/SUI/ContentView.swift

```diff
@@ -8,19 +8,158 @@
+        static let picFrame = 30
+        static let cornerRadius = 5
+        static let notifyText = "4"
+        static let emptyString = ""
+        static let appleOffersText = "Предложения Apple ID"
+        static let settingText = "Настройки"
+        static let newiOSText = "iOS 16.2: уже доступно"
+        static let airplaneImageName = "airplane"
+        static let inAirMode = "Авиарежим"
+        static let wifiImageName = "wifi"
+        static let wifiPickerText = "Wi-Fi"
+        static let bluetoothImageName = "bluetooth"
+        static let bluetoothPickerText = "Bluetooth"
+        static let antennaImageName = "antenna.radiowaves.left.and.right"
+        static let cellularPickerText = "Сотовая связь"
+        static let hotspotImageName = "personalhotspot"
+        static let hotspotModePickerText = "Режим модема"
+        static let airTagImageName = "airtag.radiowaves.forward.fill"
+        static let vpnModeToggleText = "VPN"
+        static let bluetoothImageFrameParam = 25
+    }
+    
+    @State var flightIsOn = false
+    @State var VPNIsOn = false
+    @State var wifiSection = 0
+    @State var bluetoothSection = 0
+    @State var cellularSection = 0
+    @State var modemSection = 0
+    var wifiParam = Constant.wifiParam
+    var onOfParam = Constant.onOfParam
+    
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        NavigationView {
+            Form {
+                Section {
+                    HStack {
+                        Image(Constant.avatarImageName)
+                            .resizable()
+                            .scaledToFill()
+                            .frame(width: CGFloat(Constant.avatarFrameWidth),
+                                   height: CGFloat(Constant.avatarFrameHeight))
+                            .clipShape(Circle())
+                        VStack {
+                            Text(Constant.accountName).font(.system(size: CGFloat(Constant.appleIDfontSize),weight: .semibold)).offset(x: CGFloat(Constant.accountNameOffset))
+                            Picker(selection: $wifiSection) {
+                            } label: {
+                                Text(Constant.appleIDText).font(.system(size: CGFloat(Constant.fontSize)))
+                            }.pickerStyle(.navigationLink)
+                        }
+                    }
+                    Section {
+                        ZStack {
+                            Circle().frame(width: CGFloat(Constant.picFrame), height: CGFloat(Constant.picFrame)).offset(x: CGFloat(Constant.notifyXOffset)).foregroundColor(.red)
+                            Text(Constant.notifyText).offset(x: CGFloat(Constant.notifyXOffset)).foregroundColor(.white)
+                            Text(Constant.emptyString).font(.system(size: CGFloat(Constant.fontSize)))
```

Comment:
```suggestion
                            Text(Constant.emptyString).font(.title)
```
тот же результат, но меньше текста
есть встроенные titlы разных размеров и не надо париться с поиском шрифтов

Reply:
оставлю так, на будущее учту)